front/app/components/Unauthorized/index.tsx

```diff
@@ -27,11 +27,9 @@ const Unauthorized = ({
 }: UnauthorizedProps) => {
   const theme = useTheme();
   const { formatMessage } = useIntl();
-  const { data: authUser } = useAuthUser();
+  const { data: authUser, isLoading } = useAuthUser();
 
-  const authUserPending = authUser === undefined;
```

Comment:
authUser === undefined means we're not signed in, so that shouldn't show a spinner.